packages/block-editor/src/components/inserter/search-items.js

```diff
@@ -11,13 +11,13 @@ import {
 } from 'lodash';
 
 /**
- * Converts the search term into a list of normalized terms.
+ * Sanitizes the search term string.
  *
- * @param {string} term The search term to normalize.
+ * @param {string} term The search term to santize.
  *
- * @return {string[]} The normalized list of search terms.
+ * @return {string} The sanitized search term.
  */
-export const normalizeSearchTerm = ( term = '' ) => {
+function sanitizeTerm( term = '' ) {
```

Comment:
Minor point, but this more of a normalisation than a sanitisation. This becomes a bit more noticeable when we get to the actual searching routine:

```js
const sanitizedSearchTerm = sanitizeTerm( searchTerm );
const sanitizedTitle = sanitizeTerm( title );
```

It's a little unexpected to see the title (a controlled piece of state that should already be clean) being "sanitised" next to the search term (an actually unclean user input).